driver/src/main/java/org/neo4j/driver/internal/async/pool/NettyChannelPool.java

```diff
@@ -76,4 +77,16 @@ protected ChannelFuture connectChannel( Bootstrap bootstrap )
         } );
         return channelFuture;
     }
+
+    @Override
+    public void close()
+    {
+        closed = true;
+        super.close();
```

Comment:
Don't know if `FixedChannelPool#close` is idempotent or not, but it would be better if we could make `closed` an `AtomicBoolean` and call `super.close()` only if it wasn't called before, via `closed.compareAndSet(false, true)`?